### PR TITLE
fix: Adds details about FOSSASIA to the About Us

### LINKED ISSUE #2043: About Us should contain additional information.
About Us should contain additional information such as about FOSSASIA, which made this application PSLab. There should be some information about FOSSASIA so people will be curious and search for other apps by FOSSASIA in case they like PSLab. This will increase downloads of FOSSASIA's all apps and softwares along with other benefits such as gained popularity, more job applications, more downloads, and more revenue.

### PR DESCRIPTION
**Pull request by AkshobhyaP**
Fixes #2043

**Changes**: Adds Details about FOSSASIA to the About Us Fragment

**Screenshot/s for the changes**:Screenshot_20191230-155745_PSLab
 
**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[feature.zip](https://github.com/fossasia/pslab-android/files/4009559/feature.zip)

### CODE REVIEW

**Comment by AkshobhyaP** [comment]
Hi, I am a beginner and this my first pull request. Please let me know if I can improve on anything.

**Comment by cweitat** [comment]
Doesn't justify why the description should be there to crowd the app.

**Comment by AkshobhyaP** [comment]
To give people an insight about FOSSASIA so that they try out other applications.

**Comment by ho-dor** [comment]
Not really sure whether we need to add this info

**Comment by AkshobhyaP** [comment]
Sir, the basic color theme is matching with the whole app but the background color depends on whether or not dark mode is enabled in main device settings.
Screenshot

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -477,11 +478,8 @@
     <string name="view_logged_data">View Logged Data</string>
     <string name="export_logged_data">Export Logged Data</string>
     <string name="sensor_logged_data">Sensor Logged Data</string>
-
     <string name="full">Full</string>
-
     <string name="pocket_science_logo">Pocket Science Logo</string>
-
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Stick to the changes addressed by the issue. Undo these changes.

File: app/src/main/java/io/pslab/fragment/AboutUsFragment.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 .addWebsite("https://goo.gl/forms/sHlmRAPFmzcGQ27u2", getString(R.string.nav_report))
                 .addItem(new Element(getString(R.string.version), R.drawable.ic_widgets_black_24dp))
                 .addItem(new Element(getString(R.string.flavor), R.drawable.ic_android_black_24dp))
-                .setDescription(getString(R.string.about_us_description))
+                .setDescription(getString(R.string.about_us_description)+getString(R.string.about_fossasia_description))
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/AboutUsFragment.java]: Why not creating the complete string in `strings.xml` without concatenating here?

Comment (AkshobhyaP) [reply app/src/main/java/io/pslab/fragment/AboutUsFragment.java]: I've done it this way now. I though it would be easier for future reference with two strings

File: app/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -49,8 +49,7 @@
...
-
+    <string name="about_us_description">Цель PSLab - создать устройство с открытым исходным кодом (открытое на всех уровнях), которое может быть использовано для экспериментов учителями, студентами и учеными. Наша крошечная карманная лаборатория предоставляет множество датчиков для проведения научных и инженерных экспериментов. Он обеспечивает функции многочисленных измерительных устройств, включая осциллограф, генератор сигналов, частотомер, программируемое напряжение, источник тока и регистратор данных.\n FOSSASIA - это организация, разрабатывающая приложения с открытым исходным кодом и Open Hardware совместно с мировым сообществом из Азии. Наша цель - предоставить доступ к открытым технологиям, научным приложениям и знаниям, которые улучшают жизнь людей. Мы хотим, чтобы люди могли адаптировать и изменять технологии в соответствии со своими собственными идеями и потребностями, а также проверять науку и знания с помощью подхода открытого доступа.</string>
```
Comment (cweitat) [inline app/src/main/res/values-ru/strings.xml]: you know/speak russian?

Comment (AkshobhyaP) [reply app/src/main/res/values-ru/strings.xml]: I used google translate for the translation.

**Comment by mariobehling** [comment]
We can only accept translations from native speakers to ensure quality translations. Therefore closing this.
